src/Sulu/Bundle/AdminBundle/Controller/AdminController.php

```diff
@@ -208,7 +209,17 @@ public function contextsAction(Request $request)
         $contexts = $this->adminPool->getSecurityContexts();
         $system = $request->get('system');
 
-        $response = isset($system) ? $contexts[$system] : $contexts;
+        $mappedContexts = [];
+
+        foreach ($contexts as $system => $sections) {
+            foreach ($sections as $section => $contexts) {
+                foreach ($contexts as $context => $permissionTypes) {
+                    $this->addContext($mappedContexts, $system, $section, $context, $permissionTypes);
```

Comment:
That's what I actually wanted to avoid :sweat: The initial idea doesn't work, because different keys are used... But it's not too bad, I can live with it.